prow/plugins/lgtm/lgtm.go

```diff
@@ -35,7 +36,18 @@ var (
 )
 
 func init() {
-	plugins.RegisterGenericCommentHandler(pluginName, handleGenericComment, nil)
+	plugins.RegisterGenericCommentHandler(pluginName, handleGenericComment, helpProvider)
+}
+
+func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
+	// The Config field is omitted because this plugin is not configurable.
+	return &pluginhelp.PluginHelp{
+			Description: "The lgtm plugin manages the application and removal of the 'lgtm' (Looks Good To Me) label which is typically used to gate merging.",
+			WhoCanUse:   "Only assignees can apply the 'lgtm' label, but if you are not an assignee and you use the '/lgtm' command the plugin will attempt to assign you to the pull request if you are a member of the organization that owns the repository. The '/lgtm cancel' command works similarly, but the pull request author is also allowed to use this command.",
```

Comment:
`Members of the organization that owns the repository. '/lgtm cancel' can be used additionally by the PR author`?

Reply:
Done.